sentinel-core/src/main/java/com/alibaba/csp/sentinel/config/SentinelConfig.java

```diff
@@ -34,15 +34,18 @@
  */
 public class SentinelConfig {
 
-    private static final Map<String, String> props = new ConcurrentHashMap<String, String>();
+    private static final Map<String, String> PROPS = new ConcurrentHashMap<String, String>();
 
     public static final String CHARSET = "csp.sentinel.charset";
     public static final String SINGLE_METRIC_FILE_SIZE = "csp.sentinel.metric.file.single.size";
     public static final String TOTAL_METRIC_FILE_COUNT = "csp.sentinel.metric.file.total.count";
     public static final String COLD_FACTOR = "csp.sentinel.flow.cold.factor";
+    public static final String HEARTBEAT_INTERVAL_MS = "csp.sentinel.heartbeat.interval.ms";
```

Comment:
You should not put transport related things in `sentinel-core`.

Reply:
yes, sorry, I will delete it.